contracts/interface/ILimitOrderBook.sol

```diff
@@ -99,8 +98,8 @@ interface ILimitOrderBook {
 
     /// @param order LimitOrder struct
     /// @param signature The EIP-712 signature of `order` generated from `eth_signTypedData_V4`
-    /// @param roundIdWhenTriggered The oracle `roundId` when triggerPrice is satisfied
-    // Only avaliable if orderType is StopLimitOrder, otherwise set to 0
+    /// @param roundIdWhenTriggered Chainlink `roundId` when triggerPrice is satisfied
+    // Only avaliable if orderType is StopLimitOrder/TakeProfitLimitOrder, otherwise set to 0
     function fillLimitOrder(
         LimitOrder memory order,
         bytes memory signature,
```

Comment:
Also, curious about should we place `getOrderHash` & `verifySigner` in the interface?

Reply:
It looks like the compiler only allows external functions in the interface, but `getOrderHash` is public. Is it ok to write `function getOrderHash(LimitOrder memory order) external view returns (bytes32)`?

Compiler passes though, so I added it anyway.